Directory.Build.props

```diff
@@ -5,13 +5,28 @@
     <PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
     <RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
     <RepositoryType>Git</RepositoryType>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <Copyright>Pekka Heikura</Copyright>
     <Authors>Pekka Heikura</Authors>
     <Product>graphql-dotnet server</Product>
     <Company>graphql-dotnet</Company>
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <VersionPrefix>4.1.0-preview</VersionPrefix>
+    <VersionSuffix>dev</VersionSuffix>
```

Comment:
```suggestion
```

Why?

Reply:
Local builds or otherwise where it’s not running from GitHub will create builds with a dev suffix

Reply:
Why is this needed? We didn't use it in GraphQL.NET.